libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Cert.hs

```diff
@@ -30,17 +27,25 @@ import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Deleg (nameDelegCert)
 import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.GovCert (nameGovCert)
 import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Pool (namePoolCert)
 import Test.Cardano.Ledger.Constrained.Conway
+import Test.Cardano.Ledger.Constrained.Conway.WitnessUniverse
 
 instance
-  ( IsConwayUniv fn
-  , Inject (ConwayCertExecContext Conway) (Map (RewardAccount StandardCrypto) Coin)
-  ) =>
+  IsConwayUniv fn =>
   ExecSpecRule fn "CERT" Conway
   where
-  type ExecContext fn "CERT" Conway = ConwayCertExecContext Conway
-  environmentSpec _ = certEnvSpec
-  stateSpec ctx _ = certStateSpec (lit $ ccecDelegatees ctx)
-  signalSpec _ = txCertSpec
+  type ExecContext fn "CERT" Conway = (WitUniv Conway, ConwayCertExecContext Conway)
```

Comment:
It'd be better to add a witness universe field to `ConwayCertExecContext` and then get rid of the pair here

Reply:
That requires adding a new field, and adding lots of code to instances for  ConwayCertExecContext. This is simpler and doesn't mix up the two concerns.